client/rest-high-level/src/main/java/org/elasticsearch/client/security/InvalidateTokenRequest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.security;
+
+import org.elasticsearch.client.Validatable;
+import org.elasticsearch.common.Nullable;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Request to invalidate a OAuth2 token within the Elasticsearch cluster.
+ */
+public final class InvalidateTokenRequest implements Validatable, ToXContentObject {
+
+    private final String accessToken;
+    private final String refreshToken;
+
+    InvalidateTokenRequest(@Nullable String accessToken, @Nullable String refreshToken) {
+        this.accessToken = accessToken;
```

Comment:
Technically shouldnt we have a check for both being null/empty and both being non empty?